middleware.js

```diff
@@ -49,6 +61,24 @@ module.exports.fetchRandomGameData = async (req, res, next) => {
   }
 };
 
+module.exports.fetchRandomGameDataArr = async (req, res, next) => {
+  const pageNumArr = generateUniqueRandomArr();
+  let gameDataArr = [];
+  for (let j = 0; j < 5; j++) {
+    response = await axios.get(
```

Comment:
`response` should be defined, otherwise you will get `lint` errors

Reply:
Basically what happens here is that you are declaring a global variable when you don't use a declaration keyword, try to avoid it otherwise there might be unexpected behaviors in your app.

Reply:
Thanks for pointing it out